core/src/main/scala/io/finch/EndpointModule.scala

```diff
@@ -289,6 +290,22 @@ trait EndpointModule[F[_]] {
   def asyncBody(implicit F: Effect[F]): Endpoint[F, AsyncStream[Buf]] =
```

Comment:
My questions about this:

 - Can we somehow allow users to work with `AsyncStream[Buf]` body stream via the new API (provide appropriate instances)
 - If we can, let's deprecate this method.

Reply:
I need to think about it. My gut feeling is that it's possible, but type aliases should be involved.